src/stars.css

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url(images/star.svg);
+  margin-right: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background: url("./images/star-active.svg");
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
it's redundant properties, you already define it in `.stars__star`

Reply:
Sorry , but I didn’t understand that mistake. 24 and 25 lines are responsible for position yellow stars